src/xcode/ENA.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -30,10 +30,10 @@
       },
       {
         "package": "CertLogic",
-        "repositoryURL": "https://github.com/corona-warn-app/dgc-certlogic-ios",
+        "repositoryURL": "https://github.com/eu-digital-green-certificates/dgc-certlogic-ios",
```

Comment:
Why is this part of the PR? I think you should merge 2.10 into your base branch "feature/9213-Trigger-Booster-Notifications"

Reply:
I just merged and pushed again, there were conflicts, but i didn't change this, which one is the correct URL?

Reply:
The correct ones are the ones from the 2.10 Branch.
"https://github.com/eu-digital-green-certificates/dgc-certlogic-ios"

Please crosscheck precisely.

Reply:
Yes i have the same now after i merged 2.10